go.mod

```diff
@@ -503,6 +504,7 @@ replace (
 	k8s.io/kube-scheduler => ./staging/src/k8s.io/kube-scheduler
 	k8s.io/kubectl => ./staging/src/k8s.io/kubectl
 	k8s.io/kubelet => ./staging/src/k8s.io/kubelet
+	k8s.io/kubernetes => k8s.io/kubernetes v1.23.5
```

Comment:
this stopped the other versions from getting bumped, I think the issue was that the apiextensions-apiserver imports kubernetes and the vendor script auto bumps that. I imagine this is not what we want to do but getting closer at least